docs/ops/type/Convert_1.md

```diff
@@ -4,7 +4,16 @@
 
 **Category**: type conversion
```

Comment:
I see in many PRs the category enclosed by *. Up to you, but probably starting by capital letter. eg:
\*Type conversion\*

Reply:
Done [here](https://github.com/openvinotoolkit/openvino/pull/5530/commits/f4e8c0dd48f2d34f44b7692c2426a194fa4f0852)